src/components/panel/Panel.js

```diff
@@ -105,10 +105,18 @@ export default class PanelComponent extends NestedComponent {
     let panelClass = 'mb-2 card border ';
     panelClass += `panel panel-${this.component.theme} `;
     panelClass += this.component.customClass;
-    this.element = this.ce('div', {
```

Comment:
This should be put inside of the ```this.empty``` method. And from what I understand, ```this.empty()``` does get called before a "rebuild" so maybe the issue is that a Panel is not implementing ```this.empty()``` correctly.

Reply:
this.empty() worked well (cleaned children elements), but new element attacting did not work.
If data is changed, then called Base.redraw().
https://github.com/formio/formio.js/blob/master/src/components/base/Base.js#L1476
`this.build(this.clear());`
Actions:
1. called component.empty() - clean children elements
2. called component.build() - create element only.

https://github.com/formio/formio.js/blob/master/src/components/base/Base.js#L758-L762

Base.createElement() - create or update existed element.

We have to check an element existing. If element is already created, then we have to update element.